windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec

```diff
@@ -0,0 +1,30 @@
+# react-native-windows-uwp-camera.podspec
+
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "react-native-windows-uwp-camera"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.description  = <<-DESC
+                  react-native-windows-uwp-camera
+                   DESC
+  s.homepage     = "https://github.com/github_account/react-native-windows-uwp-camera"
+  # brief license entry:
+  s.license      = "MIT"
+  # optional - use expanded license entry instead:
+  # s.license    = { :type => "MIT", :file => "LICENSE" }
+  s.authors      = { "Your Name" => "yourname@email.com" }
```

Comment:
is this podspec for local consumption or for publishing?

Reply:
This is an auto generated file for iOS. I can probably delete it

Reply:
OK, file has been deleted